common/apm/src/metrics/consensus.rs

```diff
@@ -72,6 +72,10 @@ lazy_static! {
         "The committed transactions"
     )
     .unwrap();
+    pub static ref ENGINE_ORDER_TX_GAUGE: IntGauge =
```

Comment:
Would you explain the purpose of this gauge? I'm guessing you want to be accumulating the number of `proposals_hash`s and `transaction`s received from the proposal.